app/views/taxons/sections/_news_and_communications.html.erb

```diff
@@ -3,9 +3,9 @@
   featured_item_layout_class = "taxon-page__featured-item--single" if section[:documents].empty?
 %>
 
-<div class="grid-row">
-  <div class="taxon-page__featured-item column-one-third <%= featured_item_layout_class %>" <%= "data-module=track-click" if track_click?(section[:promoted_content]) %>>
+  <div class="taxon-page__featured-item <%= featured_item_layout_class %>" <%= "data-module=track-click" if track_click?(section[:promoted_content]) %>>
     <%= render "govuk_publishing_components/components/image_card", {
+        large: true,
```

Comment:
Once [the fix](https://github.com/alphagov/collections/pull/918) is merged, you should be able to rebase to fix the issue with the news image card layout.